components/sup/src/lock_file.rs

```diff
@@ -348,6 +348,7 @@ mod tests {
               C: AsRef<str>
     {
         let mut file = std::fs::OpenOptions::new().write(true)
+                                                  .truncate(true)
```

Comment:
@mwrock should the same apply here as well?

Reply:
that one looks ok. This is a helper function for tests and all of its callers expect an empty file.

Reply:
okay